packages/react-components/src/components/Search/Search.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { ComponentMeta, Story } from '@storybook/react';
+
+import { Search as SearchComponent, ISearchProps, SearchSize } from './Search';
+
+export default {
+  title: 'Components/Search',
+  component: SearchComponent,
+} as ComponentMeta<typeof SearchComponent>;
+
+const containerStyles = { width: 300, marginBottom: 15 };
+const textStyles = { marginBottom: 5, fontSize: 15 };
+
+const StoryTemplate: Story<ISearchProps> = (args: ISearchProps) => (
+  <div style={{ width: 300 }}>
+    <SearchComponent {...args} />
+  </div>
+);
+
+export const Search = StoryTemplate.bind({});
+Search.args = {
+  onChange: (v) => console.log(v),
```

Comment:
Please don't use console.logs or alerts. The better alternative is to use `actions` https://storybook.js.org/docs/react/essentials/actions